[2021-03-22 21:15:07,506] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-03-20T00:00:00+00:00 [queued]>
[2021-03-22 21:15:07,516] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop 2021-03-20T00:00:00+00:00 [queued]>
[2021-03-22 21:15:07,517] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:07,517] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-22 21:15:07,517] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-22 21:15:07,523] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): run_after_loop> on 2021-03-20T00:00:00+00:00
[2021-03-22 21:15:07,526] {standard_task_runner.py:52} INFO - Started process 71 to run task
[2021-03-22 21:15:07,528] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', '2021-03-20T00:00:00+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp8u7xwgfb', '--error-file', '/tmp/tmpifbc53za']
[2021-03-22 21:15:07,528] {standard_task_runner.py:77} INFO - Job 11: Subtask run_after_loop
[2021-03-22 21:15:07,555] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.run_after_loop 2021-03-20T00:00:00+00:00 [running]> on host 699fbf69fb73
[2021-03-22 21:15:07,581] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2021-03-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-20T00:00:00+00:00
[2021-03-22 21:15:07,582] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-22 21:15:07,582] {bash.py:158} INFO - Running command: echo 1
[2021-03-22 21:15:07,587] {bash.py:169} INFO - Output:
[2021-03-22 21:15:07,588] {bash.py:173} INFO - 1
[2021-03-22 21:15:07,588] {bash.py:177} INFO - Command exited with return code 0
[2021-03-22 21:15:07,605] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20210320T000000, start_date=20210322T211507, end_date=20210322T211507
[2021-03-22 21:15:07,624] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-22 21:15:07,661] {local_task_job.py:146} INFO - Task exited with return code 0
